File: app/src/main/java/com/song/deviceinfo/ui/virtual/VirtualAppViewModel.java
package com.song.deviceinfo.ui.virtual;

import android.content.Context;

import com.song.deviceinfo.info.VirtualAppInfo;
import com.song.deviceinfo.ui.base.NormalViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2020/7/10.
 */
public class VirtualAppViewModel extends NormalViewModel {

    public List<Pair<String, String>> getVirtualAppInfo(Context context) {
        return VirtualAppInfo.getVirtualAppInfo(context);
    }

}

File: app/src/main/java/com/song/deviceinfo/ui/virtual/VirtualAppViewModel.kt
package com.song.deviceinfo.ui.virtual

import android.content.Context
import androidx.core.util.Pair
import com.song.deviceinfo.info.VirtualAppInfo
import com.song.deviceinfo.ui.base.NormalViewModel

/**
 * Created by chensongsong on 2020/7/10.
 */
class VirtualAppViewModel : NormalViewModel() {
    fun getVirtualAppInfo(context: Context): List<Pair<String, String>> {
        return VirtualAppInfo.getVirtualAppInfo(context)
    }
}
